src/components/theme_switcher/theme_switcher.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="qld__theme_switcher" data-component="qgds-theme_switcher">
+  <button class="theme-toggle" id="theme-toggle" title="Toggles light & dark" aria-label="auto" aria-live="polite">
```

Comment:
should we use id's (restricting us to just one of these) would it not be better to bind by data element or 'class'?